keps/0001-universal-operator.md

```diff
@@ -0,0 +1,28 @@
+# Maestro: A Universal Operator
+
+## Summary
+
+We propose a universal operator that generates and runs operators based on a standard data format. This universal operator manages the lifecycle of stateful workloads.
```

Comment:
Is it "universal" or just for stateful things?

A concrete list of operators this would and would not support would be helpful.